api/.env.development

```diff
@@ -31,4 +31,3 @@ MLP_URL=http://mlp.example.com/api/v1
 MLP_ENCRYPTION_KEY=
 
 TURING_ENCRYPTION_KEY=test_encryption_key
-SWAGGER_FILE=./swagger.yaml
```

Comment:
Why is this removed? Is it because the default value exists and this is not really needed? https://github.com/gojek/turing/blob/main/api/turing/config/config.go#L326

Reply:
It's because with recent changes from @ashwinath, we introduced support of multiple spec files, so the validation of API endpoints can be done among all of them. And with that, `SWAGGER_FILE` is no longer in use (the config key was replaced with `SwaggerFiles` of `array` type). 

But now I'm going to revert it because this proposed specs split makes it possible to construct a fairly large single spec file while keeping it maintainable.

Reply:
Reverted it now